engine/classes/ElggPluginPackage.php

```diff
@@ -286,7 +289,7 @@ private function hasSaneDependencies() {
 			// only valid provide types
 			if (!in_array($provide['type'], $this->providesSupportedTypes)) {
 				$this->errorMsg =
-					_elgg_services()->translator->translate('ElggPluginPackage:InvalidPlugin:InvalidProvides', [$provide['type']]);
+					elgg_echo('ElggPluginPackage:InvalidPlugin:InvalidProvides', [$provide['type']]);
```

Comment:
Is it possible to keep the `elgg_echo` on the same line a `$this->errorMsg`? now that the line is shorter.....